roles/agnosticv/defaults/main.yml

```diff
@@ -123,3 +123,16 @@ execution_environment: >-
 
 # Override __meta__.deployer.execution_environment
 default_execution_environment: {}
+
+
+# GitHub integration
+github_comment_success: |
+  Update by AgnosticV operator for revision {{ r_repo.after }} finished successfully.
+  The change from this PR should be included.
+
+  You can see the updated catalog at {{ catalog_url }}
+
+github_comment_failure: |
+  AgnosticV operator failed to update for revision {{ r_repo.after }}.
+
+  Check the logs or loki.
```

Comment:
@yvarbanov  do we have an URL we could include here to fast-jump to the interesting part?

Reply:
Not at the moment but we definitely should (if possible)